packages/commonjs/src/transform.js

```diff
@@ -60,12 +61,56 @@ export function hasCjsKeywords(code, ignoreGlobal) {
   return firstpass.test(code);
 }
 
+function isTrueNode(node) {
```

Comment:
I think that all of these AST helpers beg for a new utility file 😉

Reply:
Definitely. Only wondering if I should do these things in this PR or go for a separate refactoring PR, currently leaning towards the latter.

Reply:
I'm with you. This way we can review this PR without wondering what actually changed